.azure-pipelines/windows.yml

```diff
@@ -0,0 +1,48 @@
+
+parameters:
+  name: ''
+  vmImage: ''
+  matrix: []
+
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  variables:
+    DEPENDS_FIRST: numpy>=1.9.3 matplotlib>=3.1.1
+    DEPENDS_SECOND: bioread>=1.0.4
+    CHECK_TYPE: test
+  strategy:
+    matrix:
+      ${{ insert }}: ${{ parameters.matrix }}
+
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(PYTHON_VERSION)'
+        addToPath: true
+        architecture: '$(PYTHON_ARCH)'
+    - script: |
+        python -m pip install --upgrade pip setuptools>=30.3.0 wheel
+      displayName: 'Update build tools'
+    - script: |
+        python -m pip install %DEPENDS_FIRST%
+        python -m pip install %DEPENDS_SECOND%
+      displayName: 'Install dependencies'
+    - script: |
+        python -m pip install .[$(CHECK_TYPE)]
+      displayName: 'Install phys2bids'
+    - script: |
+        mkdir for_testing
+        cd for_testing
+        cp ../setup.cfg .
+        python -m pytest --doctest-modules --cov-report term-missing --cov=phys2bids --pyargs phys2bids
+      displayName: 'Run tests'
+      condition: and(succeeded(), eq(variables['CHECK_TYPE'], 'test'))
+    - script: |
```

Comment:
Maybe I don't understand it, but I don't think we need this. Should it be different from the Travis codecov anyway?

Reply:
In theory it _shouldn't_ be different but there's a possibility that different OSs could hit different lines 🤷‍♂️ 

I'm not sure I see the harm in uploading it either way! Let me know if you have a reason that you wouldn't want it uploaded.

Reply:
Well, for one I don't know how it would behave - will we have two codecov reports, or will we get the report from the pipeline that finishes last/first?
BUT, in case we want it, then I'm sorry to say that I guess you'll have to check it as it's not going through :sweat_smile:

Reply:
Are you sure? Azure _thinks_ it's going through (see [relevant log here](https://dev.azure.com/physiopy/phys2bids/_build/results?buildId=10&view=logs&j=2c855bde-037d-57e7-f7a3-61fc65066dac&t=dd71b88c-512f-5964-a9d7-f49e4ccf9e5a&l=56))... And codecov generally [merges reports together](https://docs.codecov.io/docs/merging-reports) without any issues.

Reply:
Yup, you're right, sorry!
Do you think it's feasible to:
1. Correct the errors that pop up in this [log](https://dev.azure.com/physiopy/phys2bids/_build/results?buildId=10&view=logs&j=2c855bde-037d-57e7-f7a3-61fc65066dac&t=dd71b88c-512f-5964-a9d7-f49e4ccf9e5a&l=56) and
2. Make the reports from Azure to clearly state they're from Azure, instead of being "Unknowns"?  
If not, we can merge this in and open issues about those two points for later work.